conf/messages.en

```diff
@@ -325,6 +324,11 @@ dashboard.validation.badge = {0} validation badge
 dashboard.mistakes.header = Your recent labeling mistakes
 dashboard.mistakes.tooltip = These are mistakes as determined by other users from the Validate page. Sometimes mistakes happen while validating, and there is often nuance in these evaluations.
 dashboard.mistakes.tooltip.alt = Question mark icon for helpful tooltip.
+dashboard.team.title = Create a new team
+dashboard.team.name = Team name
+dashboard.team.description = Team description (optional)
+dashboard.team.cancel = Cancel
```

Comment:
Didn't notice this last time around, but we actually have a translation for cancel, so you don't need to add one! It's worth doing a quick search through this file for a super common translation that we may have done before. It's near the top of the file, with just the key `cancel`!